source/User_Guide/Settings/subusers.md

```diff
@@ -0,0 +1,199 @@
+{% endinfo %}
+
+*To add a domain whitelabel to a Subuser account:*
+
+1. Navigate to **Settings** and select **Subuser Management**. 
+1. Locate the Subuser account you wish to delete and click on the name to view the account details. 
+1. On the *Account Settings* page, navigate to the Whitelabel Domain section and then click **Change Sending**. 
+1. From the drop-down menus that appear, select a whitelabel domain and/or a whitelabel link to associate with the Subuser account. 
+1. Once you've made all of your selections, click **Save**. 
+
+{% anchor h2 %}
+Monitoring Subuser email content
+{% endanchor %}
+
+This feature allows you to recieve a sample of a Subuser's email content to help ensure that the content is not damaging your reputation. 
+{% warning %}
+One email credit will be charged for every monitor email that is sent
+{% endwarning %}
+The “Collect Sample Rate” is the frequency is the number of emails SendGrid will allow through before copying the email address you set on the next email sent. For example, if you set the collect sample rate at 100 then you will receive a copy of every 100th email this Subuser sends.
+
+{% anchor h2 %}
+Impersonating a Subuser
+{% endanchor %}
+
+When you set up Subusers, it may be useful to see what the Subuser’s settings are and to see what the Subuser can see in the SendGrid UI.
+To make this easier, we’ve made it so you can impersonate a Subuser from your parent account and then manage the Subuser the same way as if you had logged in with that user’s credentials, without having to log out of your parent account.
+
+*To impersonate a Subuser:*
+
+1. Log into SendGrid. 
+2. Go to the top left corner of the portal screen and click on your name just above the navigation menu.
+3. Click **Switch User**. 
+4. Choose the Subuser you want to impersonate by clicking that Subuser’s name.
+
+The page will refresh and you will notice a message at the top of the screen that says, “You're currently logged in as [Subuser name]”.
+
+If you have more than one Subuser you can also switch to another Subuser by clicking **Switch Subuser** in the top-right corner once you are impersonating any Subuser.
+
+{% anchor h3 %}
+Switching back to your parent account
+{% endanchor %}
+
+Click the link at the top left that says “Back to Parent Account”. This will log you out of the Subuser account and take you back to your parent account portal.
+
+
+{% anchor h2 %}
+Export Subuser Information
+{% endanchor %}
+
+If you would like to see how your Subusers are performing in comparison to each other, you can export Subuser data. 
+
+*To export Subuser information:*
+
+1. Navigate to **Settings** and select **Subuser Management**. 
+1. Click **Export Data**. 
+ <br>This downloads a CSV file that will have each Subuser’s reputation, number of requests each Subuser made this month, and the number of requests each Subuser made last month. 
```

Comment:
This downloads a CSV file with each Subuser’s reputation, the number of requests they made this month, and the number of requests they made last month.